trace/doc.go

```diff
@@ -0,0 +1,164 @@
+// Package trace provies an API for initiating and reporting SSF
+// traces and attaching spans to them. Veneur's tracing API also
+// provides an experimental opentracing compatibility layer.
```

Comment:
I know it's implied by "experimental", but I'd recommend adding "which may be removed in a future release", because we have talked about removing it for nearly two years, and it's mostly a question of when we'll be able to shave that yak, not whether or not we actually want to.